### PR TITLE
Add Section for Series/Collections

### PR DESCRIPTION
**Pull request by cwray-tech**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description
I added a small section in this area of the docs to share some information about collections/ series. 

There is not much information in the documentation about collections or series and I felt like this would be beneficial to others looking to help with the Series feature of the website.

## Related Tickets & Documents
https://github.com/forem/forem/issues/9658

## QA Instructions, Screenshots, Recordings

_I am working on updating the series, but when I went through the docs, I didn't find any information on Series which is why I wanted to add this first._

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.forem.com
- [x] readme
- [ ] no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?
No

## [optional] What gif best describes this PR or how it makes you feel?

Happy

### CODE REVIEW

**Review by benhalpern — APPROVED** [review]

**Comment by benhalpern** [comment]
Whoops I didn't mean to approve this... Haven't fully looked into it yet.

**Comment by cwray-tech** [comment]
@benhalpern no worries. 
This was just a documentation update I did on Github. 

One issue I am having is with the Gitpod configuration. It is giving me an ActionController::InvalidAuthenticityToken error when trying to log in with the default credentials from here: https://docs.forem.com/getting-started/db/#default-admin-user

Do I have to log in via the command line in development or should I be able to log in with the form? The dump is showing an authentication token in the request.

File: docs/technical-overview/architecture.md [hunk 1]
```diff
@@ -112,6 +112,13 @@ Article has many comments and taggings through the acts-as-taggable gem, belongs
 to a single user (and possibly an organization), and is the core unit of
 content.
 
+## Collections (or series)
+
+Collections are called Series on the frontend of the website. 
+Collections group articles so that groups of articles can be read as a series of posts.
+Collections have not been fully developed as there is not the option to edit or 
+delete series/collections yet.
```
Comment (rhymes) [inline docs/technical-overview/architecture.md:120]: I don't think we should list missing features in the documentation though.

Maybe we can be a bit more colloquial as well:

`Although the source code refers to them as "collections", groups of articles are referred throughout the user interface as "series". They represent a collection of articles related to the same topic, indeed, a series`

or something like that. Deferring to @jacobherrington for the grammar check on my suggestion :P

Comment (Link2Twenty) [reply docs/technical-overview/architecture.md:120]: Close 😉

`Although the source code refers to them as "collections" groups of articles are referred to, throughout the user interface, as "series". They represent a collection of articles related to the same topic, indeed, a series`

**Comment by Link2Twenty** [comment]
@cwray-tech the credentials were wrong, I've updated the documentation now ☺️

**Review by rhymes — APPROVED** [review]

**Comment by cwray-tech** [comment]
@Link2Twenty I am still getting an error when I use that password:

ActionController::InvalidAuthenticityToken

I had tried that password earlier as well...


Here is the full error:
error

**Comment by Link2Twenty** [comment]
@cwray-tech in the .env file change APP_DOMAIN to your own Gitpod Forem URL without https (if you pop out the preview you can get the URL) and change APP_PROTOCOL to https.

You'll need to restart the pod after you save those changes but then hopefully it'll work.

**Comment by cwray-tech** [comment]
@Link2Twenty that did not work, unfortunately. I think I might open an issue for this because I saw the same thing mentioned on the website.

**Comment by Link2Twenty** [comment]
Just to clarify we're doing the same thing.

Press the pop out button
pop out preview

Get the URL of the server
address bar

Update APP_DOMAIN and APP_PROTOCOL to match the URL we got in the last step
env variables
to
env variables modified

Restart the pod in workspaces
pod workspaces

Try and log in
forem login

~Discover your name is Admin McAdmin~ And it should work
forem welcome screen

**Comment by cwray-tech** [comment]
@Link2Twenty Yes, I have tried that multiple times, and the pod just sticks at waiting for port 3000 to respond. /:

forem-forem-master (1)

**Pull request merged**